app/Controllers/userController.php

```diff
@@ -287,25 +287,29 @@ public function createAction() {
 			$new_user_name = Minz_Request::param('new_user_name');
 			$email = Minz_Request::param('new_user_email', '');
 			$passwordPlain = Minz_Request::param('new_user_passwordPlain', '', true);
+			$badRedirectUrl = [
```

Comment:
Did you check whether there is no other call to this endpoint?
Maybe it would be good to have default values `[ 'c' => 'auth', 'a' => 'register' ]` if some of the `origin*` parameters are missing

Reply:
I have not. But I'll add default values.